packages/extension-ui/src/components/PasswordField/getFeedback.ts

```diff
@@ -16,23 +16,39 @@ const APP_RELATED_INPUTS = ['aleph', 'zero', 'aleph zero', 'signer'];
 export const getUserInputs = (name?: string | null) => {
   const nameRelatedInputs = name ? [name, ...name.split(/[^a-zA-Z]+/)] : [];
 
-  return [
-    ...APP_RELATED_INPUTS,
-    ...nameRelatedInputs,
-  ];
+  return [...APP_RELATED_INPUTS, ...nameRelatedInputs];
 };
 
-export const isPasswordTooWeak = (feedback: ValidationResult) =>  {
+export const isPasswordTooWeak = (feedback: ValidationResult) => {
   return feedback.score < LOWEST_ACCEPTED_PASSWORD_SCORE;
 };
 
+const nonLatinAlphanumeric = /\W/;
+
+const addPunctuationIfMissing = (sentence: string) => {
+  const lastChar = sentence.at(-1);
+
+  if (!lastChar) {
+    return sentence;
+  }
+
+  if (lastChar.match(nonLatinAlphanumeric)) {
```

Comment:
What sentence endings other than `.`, `!` or `?` should stop us from appending a dot?

Reply:
I was a bit afraid of non-european languages in general. Perhaps it's an overkill :shrug: And basically any punctuation - but that assumes someone ends a sentance with `:` which shouldn't happen :shrug: